deploy/runtime/webapis.conf

```diff
@@ -14,7 +14,7 @@ support_email=mailto:sam.support@nrel.gov
 urdb_ask=https://en.openei.org/w/index.php?title=Special:Ask&q=<QUESTION>&po=<PROPERTIES>&eq=yes&p[format]=json
 urdb_companies_all=https://api.openei.org/utility_companies?version=3&format=json&api_key=<SAMAPIKEY>&scope=<SCOPE>
 urdb_companies_by_zip=https://developer.nrel.gov/api/utility_rates/v3.json?api_key=<SAMAPIKEY>&address=<ADDRESS>
-urdb_rates=https://api.openei.org/utility_rates?version=7&format=json&limit=<LIMIT>&detail=<DETAIL>&offset=<OFFSET>&ratesforutility=<UTILITYNAME>&getpage=<GUID>&api_key=<APIKEY>
+urdb_rates=https://api.openei.org/utility_rates?version=8&format=json&limit=<LIMIT>&detail=<DETAIL>&offset=<OFFSET>&ratesforutility=<UTILITYNAME>&getpage=<GUID>&api_key=<APIKEY>
 urdb_view_rate=https://en.openei.org/apps/IURDB
```

Comment:
I can change this in SAM-private (if you have not already) when I build the beta releases

Reply:
I did not change SAM-private. Changes to invoke.cpp will also need to be copied over to SAM-private.